frontend/src/tests/__mocks__/svg.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+// eslint-disable-next-line react/display-name
+const SvgrMock = React.forwardRef((props, ref) => <span ref={ref} {...props} />)
```

Comment:
lets change it to svg element, we will avoid changing snapshots I believe
```suggestion
const SvgrMock = React.forwardRef((props, ref) => <svg ref={ref} {...props} />)
```